src/Components/Home/Home.css

```diff
@@ -301,7 +301,7 @@ footer {
 }
 
 .card:hover{
-    box-shadow: 1px 1px 10px 1px var(--hover-dark-grey);
```

Comment:
the box shadows in general could be a lot more subtle, tip: try to refer to material boxshadow codepens.

Reply:
I think that too, right now I'm prioritizing the functionalities, but will surely work on that later as I'm a fan of subtle UI too, Thanks for the suggestions.